pom.xml

```diff
@@ -27,24 +28,43 @@
       <dependency>
         <groupId>commons-logging</groupId>
         <artifactId>commons-logging</artifactId>
-        <version>1.1.1</version>
+        <version>1.2</version>
+      </dependency>
+      <dependency>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+          <version>1.7.36</version>
+          <scope>provided</scope>
       </dependency>
```

Comment:
This might not be needed, since it is already a transitive dep of ActiveMQ.
Please remove this new dep and test again without it.